project-base/kubernetes/deployments/webserver-php-fpm.yml

```diff
@@ -39,13 +39,13 @@ spec:
                     image: $PHP_FPM_IMAGE
                     command: ["sh", "-c", "cp -r /var/www/html/. /tmp/source-codes"]
                     volumeMounts:
-                    -       name: source-codes
-                            mountPath: /tmp/source-codes
+                    -   name: source-codes
```

Comment:
here we have `dash` in the same position as `volumeMounts` but in another file there is `dash` indented
![image](https://user-images.githubusercontent.com/39240194/49019140-893d9f80-f18d-11e8-9715-1a6b90d4bea1.png)